katello-installer/katello-installer-base.spec

```diff
@@ -2,7 +2,7 @@
 %undefine scl_prefix
 %global scl_ruby /usr/bin/ruby
 
-#global prever .rc1
+# %%global prever .rc1
```

Comment:
I got that from here:  http://rpm.org/max-rpm-snapshot/ch-rpm-inside.html

doesn't appear that `#% %global` works according to rpmlint